stack/cas/castext2/blocks/foreach.block.php

```diff
@@ -89,8 +91,12 @@ public function compile($format, $options): ?MP_Node {
             foreach ($this->params as $key => $value) {
                 $definedbody->items[] = new MP_Operation(':', new MP_Identifier($key),
                     new MP_Indexing(new MP_Identifier('__ct2_foreach___' . $key),
-                    [new MP_List([new MP_FunctionCall(new MP_Identifier('ev'),
-                    [new MP_Identifier('__ct2_foreach___iter'), new MP_Identifier('simp')])])]));
+                    [
+                        new MP_List([
+                            new MP_FunctionCall(new MP_Identifier('ev'),
+                    [new MP_Identifier('__ct2_foreach___iter'), new MP_Identifier('simp')]),
```

Comment:
Not sure about the indentation here

Reply:
Yep, the array 'Inception' should be clearer here.